functions/ColorContrast.pq

```diff
@@ -0,0 +1,18 @@
+(lighter_color as text, darker_color as text) =>
+let
+    luminosity = (color as text, optional gamma as number) =>
+        let
+            gamma_use = if gamma = null then 2.2 else gamma,
+            luminosity = 
+                0.2126 * Number.Power(Expression.Evaluate("0x" & Text.Middle(color, 1, 2))/255,gamma_use) + // Red
+                0.7152 * Number.Power(Expression.Evaluate("0x" & Text.Middle(color, 3, 2))/255,gamma_use) + // Green
+                0.0722 * Number.Power(Expression.Evaluate("0x" & Text.Middle(color, 5, 2))/255,gamma_use),  // Blue
+        in
+            luminosity,
+    color_contrast = 
```

Comment:
Needs comments to explain why this is commented out.

Reply:
Uncommented, because ultimately, the values for the calculation are arranged according to their luminosities. lighter_ and darker_ are just used for convenience in coding.